gen/input/src/main/scala/test/StarWarsSubquery.scala

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+// format: off
+/*
+  rules = [GraphQLGen]
+  GraphQLGen.schemaDirs = ["gen/input/src/main/resources/graphql/schemas"]
+ */
+package test
+
+import clue.GraphQLSubquery
+import test.StarWars
+
+@clue.annotation.GraphQLSubquery
```

Comment:
Can we reuse the `@GraphQL` annotation and detect that the trait is extending `GraphQLSubquery`?

Reply:
I need to reload this design choice into my brain. That's the approach I started with, but I must have ran into enough problem that I changed course. Let me take another look at it.

Reply:
Thanks, just needed another try at this :)